src/parse/__tests__/td1.test.ts

```diff
@@ -71,9 +71,9 @@ describe('parse TD1', () => {
       compositeCheckDigit: '1',
     });
     expect(result.valid).toBe(false);
-    expect(result.details.find((a) => a.field === 'issuingState').valid).toBe(
-      false,
-    );
+    expect(
+      result.details.find((a) => a.field && a.field === 'issuingState')?.valid,
+    ).toBe(false);
```

Comment:
this change is suspicious

Reply:
it's just because field can be `null`
also find can return undefined so i added `?` to fix the error

Reply:
can you point me to the type definition of the result? I didn't find it when I looked at the code.